### ISSUE TITLE
Add MetaData for Blog Post

### ISSUE THREAD

**Issue by honeyhexa-tech** [honeyhexa/blogpress---template#6]
**Is your feature request related to a problem? Please describe.**
Template lacks metadata, which is affecting both search engine optimization (SEO) and accessibility. 

**Describe the solution you'd like**
Modern websites use metadata to provide search engines and users with essential information about the content of web pages. This information includes title, description, keywords, and other relevant details. Additionally, metadata also plays a significant role in ensuring the website is accessible to users with disabilities by providing context and information about the page structure.
